bin/i18n-update.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+# 
+# Handles the process of updating the i18n localizations of plugins, including Gutenberg.
+# The main goals of this command are:
+#
+#   1. Download translation files and optimize them by filtering out the unused strings,
+#      previously extracted from the source map files of React Native bundle.
+#
+#      This step produces the following output files:
+#      - src/i18n-cache/{PLUGIN_NAME}/data/{LOCALE}.json     [Translation files]
+#      - src/i18n-cache/{PLUGIN_NAME}/data/index.js          [JS file to import translations]
+#
+#   2. Generate localization strings files that include the strings only used in native JS
+#      source code files ("native.js", "ios.js or "android.js"). The translations of these
+#      strings are not included in the GlotPress projects of their plugins hence, they require
+#      to be requested as part of the main apps’ strings.
+#
+#      This step produces the following output files:
+#      - bundle/android/strings.xml                     [Localization strings files for Android platform]
+#      - bundle/ios/GutenbergNativeTranslations.swift   [Localization strings files for iOS platform]
+
+# Exit if any command fails
+set -e
+
+# Get arguments
+while test $# -gt 0; do
+  case "$1" in
+    -h|--help)
+      echo "options:"
+      echo "-h, --help    show brief help"
+      echo "-p, --path    local path for generating files (by default a temp folder will be used)"
+      echo "-d, --debug   print extra info for debugging"
+      exit 0
+      ;;
+    -p|--path*)
+      shift
+      LOCAL_PATH=$1
+      shift
+      ;;
+    -d|--debug*)
+      shift
+      DEBUG='true'
+      ;;
+    *)
+      break
+      ;;
+  esac
+done
+
+function error() {
+  echo -e "\033[0;31m$1\033[0m"
+  exit 1
+}
+
+function fetch_translations() {
+  local PLUGIN_NAME=$1
```

Comment:
minor nitpick: local variables should be lower case. ALL_UPPERCASE is generally used for exported variables only.

Reply:
I've renamed all local variables in https://github.com/wordpress-mobile/gutenberg-mobile/pull/4366/commits/9274f89558f303ea360545d9e170ba0c53489a20. Additionally, I updated the script `generate-pot-files.sh` in Gutenberg with the same format ([reference](https://github.com/WordPress/gutenberg/pull/37482/commits/f60f993c4476eb5174eb4525f3ae0e7cd7723af3)).